js/styles.js

```diff
@@ -2,26 +2,26 @@
 var stateStyle = {
   nationView: {
     active: {
-      'fill': '#BEBEBE',
-      'stroke': 'transparent', // looks OK white, too
-      'stroke-width': 0
+      'fill': 'white',
+      'stroke': 'black', // looks OK white, too
+      'stroke-width': 1.5
     },
     inactive: {
-      'fill': '#DCDCDC',
-      'stroke': 'transparent', // i think we're avoiding borders usually?
-      'stroke-width': 0
+      'fill': 'white',
+      'stroke': 'black', // i think we're avoiding borders usually?
+      'stroke-width': 1.0
     }
   },
   stateView: {
     active: {
       'fill': '#DCDCDC',
-      'stroke': 'transparent', // no need for border when there's fill
-      'stroke-width': 0
+      'stroke': 'black', // no need for border when there's fill
```

Comment:
could you take out the comments about avoiding borders now that we're not avoiding them?

Reply:
also, looks from your screenshot like state borders are coming through as grey despite being told to be black here. do you think that could be because county borders are plotted on top of state borders?

Reply:
Yeah, there are a couple of funky things going on w county borders and I was going to keep it on the back burner to see what we are going to do about z index. One thing is I think the counties appear over the state border, so they cover up the black border. Other issue I can into is if you load the page on a state zoom, the counties don’t come in until you zoom out to USA and then back in. I didn’t want to get too in the weeds on those issues since I didn’t know if there would be code changes related to the mouse overs (and related to load times waiting on the county data to come in).

Reply:
Excuse my typos and bad grammar. Am on mobile at the airport :)